### PR TITLE
Fix for new riot API

### PR DESCRIPTION
**Pull request by dleiferives**
There was an error with passing in the region.
There was a type based error since the types that were being passed in changed, and the types that were returned changed. 
All data returns same.

### CODE REVIEW

File: lolexport/export.py [hunk 1]
```diff
@@ -60,3 +67,29 @@ def export_data(api_key: str, summoner_name: str, region: str) -> List[Json]:
         m["matchData"] = get_match_data(lol_watcher, region, m["gameId"])
 
     return matches
+
+
+def fix_region(region):
```
Comment (purarue) [inline lolexport/export.py:72]: could you `region.upper()` this at the top? incase the user passes `na1` instead of `NA1`

**Comment by purarue** [comment]
Wow, thanks a bunch -- I tried to fix this a few weeks ago but hit a wall somewhere, very much appreciated 🧡

File: lolexport/export.py (hunk 1 above)
Comment (purarue) [reply lolexport/export.py:72]: Or at least while comparing, should return the original region if it doesnt match one of the items here

```python
def fix_region()...
    region_upper = region.upper()
    if region_upper == "NA1":
        ....
    return region
```

Comment (dleiferives) [reply lolexport/export.py:72]: yeah for sure

**Comment by purarue** [comment]
just as an FYI, pushed some changes to match the new API version -- is a breaking change between v4 (what this was before) and v5

Once I've finished [merging the two together](https://github.com/seanbreckenridge/lolexport/issues/5), will push a release to PyPI

Thanks again

**Comment by purarue** [comment]
added a merge command to handle combining v4/v5 and pushed a new version

**Pull request merged**